src/backend/tools/google_drive.py

```diff
@@ -63,7 +69,7 @@ def call(self, parameters: dict, **kwargs: Any) -> List[Dict[str, Any]]:
         return [dict({"text": item["name"]}) for item in items]
 
 
-class GoogleDriveAuth(BaseAuth):
+class GoogleDriveAuth(BaseToolAuthentication):
```

Comment:
This change looks good. Could you please clarify what the GOOGLE_DRIVE_TOOL_ID constant is below?